pandas/tests/series/methods/test_to_frame.py

```diff
@@ -54,3 +54,15 @@ class SubclassedFrame(DataFrame):
         assert isinstance(result, SubclassedFrame)
         expected = SubclassedFrame({"X": [1, 2, 3]})
         tm.assert_frame_equal(result, expected)
+
+    def test_to_frame_finalize(self, datetime_series):
+        # GH#28283 Call __finalize__
+        expected_attrs = {"a": 1}
+
+        datetime_series.attrs.update(expected_attrs)
+
+        df = datetime_series.to_frame()
+        assert df.attrs == expected_attrs
+
+        df = datetime_series.to_frame(name="X")
```

Comment:
Could you `pytest.mark.parameterize` on the `name` instead?

Reply:
I believe I'm missing what this might accomplish. If I add `parameterize` then are you suggesting a singular parameter or adding more names?

Reply:
For example `@pytest.mark.parameterize("name", [None, "X"])`

Reply:
I can be dense. I see it now. Thanks!